### PR TITLE
Fix event the timestamp is < unknown >

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind bug
> /kind cleanup

**What this PR does / why we need it**:
> get
image
image

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
Fix event the timestamp is < unknown >
```

### CODE REVIEW

**Review by dixudx — COMMENTED** [review]
Would be better if tests are added.

**Comment by ZP-AlwaysWin** [comment]
> Would be better if tests are added.

hi @dixudx  tests has been added

**Comment by ZP-AlwaysWin** [comment]
> Would be better if tests are added.

hi @dixudx  tests has been added

**Review by dixudx — APPROVED** [review]
/lgtm

File: pkg/printers/internalversion/printers.go [hunk 1]
```diff
@@ -1659,10 +1659,18 @@ func printEvent(obj *api.Event, options printers.GenerateOptions) ([]metav1.Tabl
 		Object: runtime.RawExtension{Object: obj},
 	}
 
-	firstTimestamp := translateTimestampSince(obj.FirstTimestamp)
-	lastTimestamp := translateTimestampSince(obj.LastTimestamp)
+	var firstTimestamp string
+	var lastTimestamp string
+	if obj.FirstTimestamp.IsZero() {
+		firstTimestamp = translateTimestampSince(obj.CreationTimestamp)
```
Comment (Pingan2017) [inline pkg/printers/internalversion/printers.go:1665]: use `eventTime` better. 
 In v1beta1.event group, `eventTime` replaces `firstTimestamp`

**Pull request closed without merging**